app/cdap/components/GridTable/index.tsx

```diff
@@ -55,6 +56,7 @@ export default function GridTable() {
       count: '0',
     },
   ]);
+  const [openColumnView, setOpenColumnView] = useState<boolean>(false);
```

Comment:
i think there should be better names for this state, plz suggest and update here